static/docs/tutorial/define-ml-pipeline.md

```diff
@@ -118,6 +118,9 @@ and Windows. Some implementation details (like inodes) might differ, but the
 overall DVC behavior is the same. See also [DVC Files and
 Directories](/doc/user-guide/dvc-files-and-directories)
 
+> Note: to prove that no duplication occurs, we could use df utility to see that 
```

Comment:
we need to mention that we need to refer to df only in case of reflinks.

Reply:
I would also put this note above the example, so that a user can see it in advance